.github/workflows/build-cli.yml

```diff
@@ -30,6 +31,103 @@ jobs:
           VERSION=$(grep -o '^version = ".*"' crates/bws/Cargo.toml | grep -Eo "[0-9]+\.[0-9]+\.[0-9]+")
           echo "package_version=$VERSION" >> $GITHUB_OUTPUT
 
+      - name: Sign if branch is main or rc
+        id: sign
+        run: |
+          if [[ $GITHUB_REF == refs/heads/main || $GITHUB_REF == refs/heads/rc ]]; then
```

Comment:
Should we also be checking for `hotfix-rc`?

Reply:
We might also want to check if the origin is actually this repo and not a fork.

```if: github.repository_owner == 'Upvision'```

Reply:
@Hinton That's great idea!